src/pages/build/ink-kit.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Ink Kit
```

Comment:
I don't think this content should be here, we'll always have a hard time syncing this with the actual documentation from the repo.

Remove all the `install`, `usage` stuff and just tell the users to check the repo

Reply:
That's a good point. Will merge with it for now and start a thread to figure out what content we'll actually put here.